src/main/java/com/guysfromusa/carsgame/v1/converters/MovementsHistoryConverter.java

```diff
@@ -0,0 +1,28 @@
+package com.guysfromusa.carsgame.v1.converters;
+
+import com.guysfromusa.carsgame.entities.MovementsHistoryEntity;
+import com.guysfromusa.carsgame.v1.model.MovementHistory;
+import com.guysfromusa.carsgame.v1.model.Point;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+import static com.google.common.collect.Lists.transform;
+import static lombok.AccessLevel.PRIVATE;
+
+@NoArgsConstructor(access = PRIVATE)
+public class MovementsHistoryConverter {
+
+    public static MovementHistory toMovementHisotry(MovementsHistoryEntity entity){
+        MovementHistory movementHistory = new MovementHistory();
```

Comment:
quite heavy query (loading car and game for each row), what about adding: //todo consider projection to avoid eagerly loaded entities